build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/shared-configuration.kt

```diff
@@ -174,9 +174,18 @@ fun Project.createTestTask(name: String, executer: String, sourceSet: SourceSet,
             jvmArgumentProviders.add(SamplesBaseDirPropertyProvider(samplesDir))
         }
         setUpAgentIfNeeded(testType, executer)
+        disableIfNeeded(testType, executer)
     }
 
 
+internal
+fun IntegrationTest.disableIfNeeded(testType: TestType, executer: String) {
+    if (testType == TestType.INTEGRATION && executer == "isolatedProjects") {
```

Comment:
Strictly speaking, check for testType is not required here, since `isolatedProjects` executer available only in integ tests, but doing so we are making this function more "precise"